cli/command/system/info.go

```diff
@@ -235,6 +235,7 @@ func printSwarmInfo(dockerCli command.Cli, info types.Info) {
 			fmt.Fprintln(dockerCli.Out(), " Default Address Pool:", strAddrPool.String())
 			fmt.Fprintln(dockerCli.Out(), " SubnetSize:", info.Swarm.Cluster.SubnetSize)
 		}
+		fmt.Fprintln(dockerCli.Out(), " Data Path Port:", info.Swarm.Cluster.DataPathPort)
```

Comment:
We should only print this if `info.Swarm.Cluster.DataPathPort > 0`, otherwise it's printing incorrect information on older daemons (which don't return the port number)